coda-src/venus/vol_cml.cc

```diff
@@ -2793,6 +2793,11 @@ void cmlent::commit(ViceVersionVector *UpdateSet)
             f->stat.VV.StoreId = sid;
             AddVVs(&f->stat.VV, UpdateSet);
         }
+
+        if (vol->IsNonReplicated()) {
```

Comment:
should this be an `else if`? Because we also call `AddVVs()` when this is the last CML entry.

Reply:
Ah, just noticed you had already fixed that in a later commit.